### ISSUE TITLE
[djs13] BassBoost command gives an error.

### ISSUE THREAD

**Issue by thetechguy-cpu** [Allvaa/lavalink-musicbot#154]
Hi there.
When attempting to use the bassboost command in the discord.js V13 branch, I get this error.

```
/root/dev/devmusicbot/devbot1/node_modules/discord.js/src/rest/RequestHandler.js:298
      throw new DiscordAPIError(data, res.status, request);
            ^

DiscordAPIError: Cannot send an empty message
    at RequestHandler.execute (/root/dev/devmusicbot/devbot1/node_modules/discord.js/src/rest/RequestHandler.js:298:13)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async RequestHandler.push (/root/dev/devmusicbot/devbot1/node_modules/discord.js/src/rest/RequestHandler.js:50:14)
    at async TextChannel.send (/root/dev/devmusicbot/devbot1/node_modules/discord.js/src/structures/interfaces/TextBasedChannel.js:171:15) {
  method: 'post',
  path: '/channels/874958294026952704/messages',
  code: 50006,
  httpStatus: 400,
  requestData: {
    json: {
      content: undefined,
      tts: false,
      nonce: undefined,
      embeds: undefined,
      components: undefined,
      username: undefined,
      avatar_url: undefined,
      allowed_mentions: { parse: [ 'users', 'roles' ], replied_user: undefined },
      flags: 0,
      message_reference: { message_id: '893486021407281153', fail_if_not_exists: true },
      attachments: undefined,
      sticker_ids: undefined
    },
    files: []
  }
}
```

How can I fix this error? It seems that the bot is trying to send an empty message?

System info:

- OS: Ubuntu 20.04.
- Node version: 16.10.0.
- Git commit: 5cdbdd26d26733dcb1c5dbb11878472077704e99

**Comment by yofukashino** [comment]
Well that's because the embed is specified as ctx.respond(embed) instead of ctx.respond({embeds:[embed]})

**Comment by Allvaa** [comment]
can you tell which command did you run? i probably missed something and I don't realize it

**Comment by yofukashino** [comment]
> can you tell which command did you run? i probably missed something and I don't realize it

read the comment above thats the mistake

**Comment by Allvaa** [comment]
> read the comment above thats the mistake

sorry, i didnt read the title, feels so stupid
just merged #155 which fix embed sending
